README.md

```diff
@@ -0,0 +1,76 @@
+# Cognoma core-service
+
+This repository, under the umbrella of Project Cognoma
+(https://github.com/cognoma), holds the source code, under open source
+license, of the Terraform configuration files used to manage the infrastructure
+for the backend of the Project Cognoma.
+
+## Getting started
+
+Make sure to fork [this repository on
+ +GitHub](https://github.com/cognoma/infrastructure "cognoma/infrastructure on
```

Comment:
These plus signs to continue the hyperlink across newlines are not part of GitHub flavored markdown. i.e. they show up in the GitHub rendering. My personal view is that it's better to put newlines after each sentence in git tracked markdown docs rather than wrap at a character cutoff.

However, I think it will work if you just delete the `+`.